modules/client-configuration/pages/snippets/trust_gpg.adoc

```diff
@@ -1,18 +1,61 @@
-By default, operating systems trust only their own GPG keys when they are installed, and do not trust keys provided by third party packages.
+Operating Systems either trust their own GPG keys directly or at least ship them installed with the minimal system.
+But third party packages signed by a different GPG key need manual handling.
 The clients can be successfully bootstrapped without the GPG key being trusted.
 However, you cannot install new client tool packages or update them until the keys are trusted.
 
-Salt clients are set to trust {suse} tools channels GPG keys when they are bootstrapped.
-For all other clients and channels, you need to manually trust third party GPG keys.
+Salt Clients use now GPG key information entered for a Software Channel to manage the trusted keys.
+When a Software Channel with GPG key information is assigned to a client, the key gets trusted
+as soon as the channel is refreshed or the first package gets installed from this channel.
 
+The GPG key url which is set of a Software Channel must exist. In case it is a file url,
+the GPG key file must be deployed on the client before the software channel is used.
```

Comment:
```suggestion
// The meaning of the following sentence is not clear
The GPG key URL which is set of a software channel must exist.
In case it is a file URL, the GPG key file must be deployed on the client before the software channel is used.
```